jekyll/_includes/snippets/custom-webhook-setup.adoc

```diff
@@ -11,11 +11,15 @@ NOTE: Selecting a custom webhook as a pipeline trigger is generally available fo
 . Select btn:[Next].
 +
 image::triggers/custom-webhooks-add-trigger.png[screenshot showing adding a custom webhook trigger]
+#rosie-to-update-image#
```

Comment:
for reference, this is the image needed: 
<img width="1046" alt="Screenshot 2024-10-14 at 19 02 20" src="https://github.com/user-attachments/assets/4282f2f4-ca58-4604-8281-5094bd7650e7">

page is live!